src/components/HURUmap/Panel/ProfileItems.js

```diff
@@ -64,53 +69,67 @@ const ProfileItems = memo(
                   id={slugify(`${category.title}-${child.title}`)}
                   title={child.title}
                 />
-                {child?.metrics?.map(
-                  ({ label, parentMetric, ...other }, metricIndex) => {
-                    const secondaryMetric = getSecondaryMetric(
-                      categoryIndex,
-                      subcategoryIndex,
-                      metricIndex
-                    );
-                    return (
-                      <div key={label} className={classes.metricRow}>
-                        <KeyMetric
-                          title={label}
-                          formattedValue={formatNumericalValue(other)}
-                          parentFormattedValue={
-                            parentMetric
-                              ? formatNumericalValue(parentMetric)
-                              : undefined
-                          }
-                          {...other}
-                          color="primary"
-                          className={clsx({
-                            [classes.metric]: secondaryProfile,
+                <Grid container>
+                  {child?.metrics?.map(
```

Comment:
I kinda remember you mentioning in the morning something about these being up to 4. Not sure if you meant we should be showing 2 rows if we get _multiples_ of 4 or maximum should be 4, 2 rows.

Reply:
@kilemensi No. I said a row of 4 column grids. So in a row, according to design we need to show 3 metrics (for single profile)